resources/analysisTools/roddyTools/wrapInScript.sh

```diff
@@ -22,6 +24,66 @@ fi
 # Cluster options (like i.e. PBS ) have to be parsed and set before job submission!
 # They will be ignored after the script is wrapped.
 
+
```

Comment:
That's all in the wrapper script now, maybe you could move it to a different file?

Reply:
Aren't these functions already available in other scripts?

Reply:
Yes, they are from the workflow library. We could move them into a general workflow library in the default plugin and expose this into the other plugins. Here the code is used in a very early stage of the script, where nothing more is known, but that the wrapper was loaded. Therefore I tended to put the code here. But if we can be sure that a DefaultPlugin workflowLib.sh is equally available, we can source it here.

Reply:
If you put the script in the same directory as the wrapper, you can assume that it is this way. Only the name of the script will not be passed by Roddy. But I think in this case it is legitimate.

Reply:
You mean, we don't define a tool in the DefaultPlugin XML?